modules/appeals_api/config/schemas/v2/10182_with_shared_refs.json

```diff
@@ -1,122 +1,108 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
   "description": "JSON Schema for VA Form 10182",
-  "$ref": "#/definitions/nodCreate",
```

Comment:
This schema is used in example responses, but the validator was looking at refs across the entire doc file & complaining this path didn't exist. Decided to just peel off some onion layers of the definitions and use properties directly. The nodCreate schema is now created from rswag_config.